src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  bookItems: [],
+};
+
+const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addItem: (state, action) => {
+      const newBook = action.payloadl;
+      state.bookItems.addItem(newBook);
+    },
+    removeBook: (state, action) => {
+      const bookName = action.payload;
+      state.bookItems = state.bookItems.filter((item) => item.bname !== bookName);
+    },
+  },
```

Comment:
- [ ] As required for the project to defire a `slice` of state for books that includes an array of books (initial state: empty array), a reducer that adds a book, a reducer that removes a book, I think it would be better if you update the above code lines as they contains some typing errors(`payloadl` instead of `payload`), you are sung names of parameters which don't exist,... I recoomend you update it the below code(for both `addBook` and `removeBook` reducers):
```javascript
export const booksSlice = createSlice({
  name: "books",
  initialState,
  reducers: {
    addBook: (state, action) => {
      state.books = [
        ...state.bookItems,
        {
          id: Math.floor(Math.random() * 100), // You can also use the Uuid npm to generate unique id 
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
    removeBook: (state, action) => {
      state.books = state.bookItems.filter((book) => book.id !== action.id)
    },
  },
})
```